bash-preexec.sh

```diff
@@ -310,6 +320,7 @@ __bp_install() {
     local existing_prompt_command
     # Remove setting our trap install string and sanitize the existing prompt command string
     existing_prompt_command="${PROMPT_COMMAND:-}"
+	# shellcheck disable=SC1087
     existing_prompt_command="${existing_prompt_command//$__bp_install_string[;$'\n']}" # Edge case of appending to PROMPT_COMMAND
```

Comment:
I think what `shellcheck` recommends is this:

```suggestion
    existing_prompt_command="${existing_prompt_command//${__bp_install_string}[;$'\n']}" # Edge case of appending to PROMPT_COMMAND
```

Reply:
Makes sense, agree.

Reply:
> you can't nest curly braces like that,

What do you mean? We can do that.

> the maintainer can update the code if they feel it is needed.

Right. The maintainer can finally decide whether we ignore the warning or follow the suggestion.

Reply:
In fact you are right, 
TIL that you can use bracketed variable names inside a parameter expansion. 
OK Thanks for the new knowledge.

Reply:
We can put through this PR as is for now. 

I'm in favor of the update though vs disabling the warning. Feel free to submit this update as a small PR as well if it is essentially a no-op.